app/src/main/java/com/mobile/tandil/javabaseproject/mvp/view/ParkingView.java

```diff
@@ -29,4 +34,22 @@ public void showDialogFragment(ListenerDialogFragment listenerDialogFragment) {
     public void showNumberOfParking(String numberOfParking) {
         binding.textViewMainActivity.setText(numberOfParking);
     }
+
+    @Override
+    public void showReservationActivity() {
+        Context context = getContext();
```

Comment:
There's no need to make getContext and getActivity, because Activity extends from Context

Reply:
done